pcsx2/CDVD/CDVD.cpp

```diff
@@ -958,7 +958,7 @@ u8 monthmap[13] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
 
 void cdvdVsync() {
 	cdvd.RTCcount++;
-	if (cdvd.RTCcount < ((gsRegionMode == Region_NTSC) ? 60 : 50)) return;
+	if (cdvd.RTCcount < ((gsVideoMode == NTSC) ? 60 : 50)) return;
```

Comment:
then current implementation has a bug :P  It'll use 50 for progressive mode whereas it ought to use 60.